content/en/basics/interplanetary-consensus/overview/index.md

```diff
@@ -13,45 +13,38 @@ weight: 100
 toc: true
 ---
 
-## The problem with consensus
-
 Many blockchain networks, including Filecoin, require all validators to process _all_ transactions. This causes a major bottleneck for the network and makes it difficult to increase network performance by scaling out. On top of that, different applications have different performance and security requirements, making it difficult for a single consensus layer to accommodate any type of web3 application.
 
-Interplanetary Consensus (IPC) is a framework to enable on-demand, horizontal scalability of Filecoin by deploying subnets that spawn their own state, validate messages in parallel, and seamlessly interact with any network in the hierarchy, as well as with the Filecoin root network. Subnets can run different consensus algorithms depending on application requirements.
+Interplanetary Consensus (IPC) is a framework to enable on-demand, horizontal scalability of Filecoin by deploying subnets that spawn their own state, validate messages in parallel, and seamlessly interact with any network in the hierarchy, as well as with the Filecoin root network. Subnets can run different consensus algorithms depending on application requirements, and are particularly well suited to scenarios requiring fast finality.
```

Comment:
```suggestion
Interplanetary Consensus (IPC) is a framework that enables on-demand, horizontal scalability of Filecoin by:

- Deploying subnets that spawn their own state.
- Validating messages in parallel.
- Seamlessly interacting with any network in the hierarchy, as well as with the Filecoin root network. 

Subnets can run different consensus algorithms depending on application requirements, and are particularly well suited to scenarios requiring fast finality.
```

Reply:
```suggestion
Interplanetary Consensus (IPC) is a framework that enables on-demand, horizontal scalability of Filecoin by deploying subnets that:

- Spawn their own state.
- Validate messages in parallel.
- Interact with any other subnet in the hierarchy, as well as with the Filecoin root network. 

Subnets can run different consensus algorithms depending on application requirements, and are particularly well suited to scenarios requiring fast finality.
```

Your suggestion with one tweak: it's the subnets that validate messages in parallel and seamless interact. It reads a bit weird when applied to Filecoin, even if you can get to the meaning. My original wording was also not great at times, so I fixed that too.